lib/vanagon/driver.rb

```diff
@@ -77,9 +77,43 @@ def build_host_info
       { "name" => @engine.build_host_name, "engine" => @engine.name }
     end
 
-    # Returns the set difference between the build_requires and the components to get a list of external dependencies that need to be installed.
+    # Normalize our pkgname=version syntax to work with the package
+    # manager being used for this platform.
+    def normalize_build_dependencies_syntax(build_deps)
+      if @platform.build_dependencies.command =~ /apt/
+        # apt uses '=' as the package/version separator, no need to do anything
+      elsif @platform.build_dependencies.command =~ /yum|dnf|zypper/
+        # yum/dnf/zypper uses '-' as the package/version separator
+        build_deps = build_deps.map { |pkg| pkg.sub('=', '-') }
+      else
```

Comment:
This may not be feasible due to how chocolaty passes in the version of a package to install. I can't remember off the top of my head though